src/storyshots.test.js

```diff
@@ -24,6 +26,25 @@ jest.mock('./util', () => ({
 
 jest.mock('react-draggable', () => 'Draggable');
 
+// Mock context value
+const mockContextValue = {
+  meetingPinPasswd: '123456',
+  participantName: 'Alice',
+  setMeetingPinPasswd: (value) => {
+    mockContextValue.meetingPinPasswd = value;
+  },
+  setParticipantName: (value) => {
+    mockContextValue.participantName = value;
+  },
+};
+
+// Wrap story rendering with context
+const wrapWithContext = (story) => (
```

Comment:
We need to wrap the `MeetingContext` before render so that the mock values can be found in the storyshot tests. Without this fix, all WebexMeeting related story tests were failing.